Orange/widgets/data/owtable.py

```diff
@@ -509,7 +515,7 @@ def __restore_sort(self) -> None:
         assert self.input is not None
         sort = self.__pending_sort
         self.__pending_sort = None
-        if sort is None:
+        if not sort:
```

Comment:
Here, it happenned that `sort` was `[]`, not `None`. The remaining code had no effect (as far as I see), but caused the button to be enabled. I could use `self.restore_button.setEnabled(bool(sort))` below, but I think this is a better fix.